crates/bevy_input/src/mouse.rs

```diff
@@ -155,3 +156,79 @@ pub fn mouse_button_input_system(
         }
     }
 }
+
+/// Tracks how much the mouse has moved every frame
+/// This resource is reset to zero every frame
+///
+/// This resource sums the total [`MouseMotion`] events received this frame.
+#[derive(Resource, Debug, Clone, Copy, PartialEq, Reflect, Default)]
+#[reflect(Debug, PartialEq)]
```

Comment:
These types should probably also register `ReflectDefault` (i.e. `#[reflect(Default)]`)